wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights All Time: update to new design

Fixes #11906 

This adds the structure for two-column stat cards, and utilizes it for Insights _All-Time_.
- `StatsTwoColumnRow` - shows a single data row in two columns.
- `TwoColumnCell`:
  - Adds the required number of `StatsTwoColumnRow`s to a `UIStackView`.
  - Shows a `View more` options when necessary. (It is _not_ shown for _All-Time_.)
- `TwoColumnCell` is now used to display _All-Time_ stats.

To test:

- On a site with no All Time data, go to Insights _All-Time_.
  - Verify `No data yet` is shown.
- On a site with All Time data, go to Insights _All-Time_.
  - Verify the data is shown in two columns.
  - Verify `View more` is not shown.

Update release notes:
